### DISCUSSION TITLE
Rightclick bag from anywhere in inventory to open.

### DISCUSSION THREAD

**Discussion by JasonDayspring** [P3pp3rF1y/SophisticatedBackpacks#1120] [Feature / Idea Archive]
That's it.  I just updated from 1.12.2, and this has amazing features I never dreamed of, but I can't just open the bags from my inventory.  I find it strange that none of the backpack mods available for 1.21.1 can do that.  (I just finished testing the last one, it was so ugly that I was a bit relieved I wasn't stuck using it.)  Is there a technical reason why that isn't possible anymore?

**Comment by P3pp3rF1y** [comment]
this is a feature just open backpack keybind while hovering it. Also please don't repost the same message in multiple places
